regtest.sh

```diff
@@ -240,6 +240,8 @@ make-summary() {
     | sed -e '/TABLE_ROWS/ r rows.html' \
     > $filename
 
+  rm rows.html
```

Comment:
I removed the line for now, but still wonder - why do we need the rows.html file lying around?